tmd/Topology/analysis.py

```diff
@@ -20,6 +20,67 @@ def sort_ph(ph):
     return np.array(ph)[np.argsort([p[0] - p[1] for p in ph])].tolist()
 
 
+def closest_ph(ph_list, target_extent, method='from_above', return_index=False):
```

Comment:
Better to define an enum for the different method values instead of using strings

Reply:
For such a small function?

Reply:
Ok, maybe not